File: Python Projects/Ex_Files_Python_Basics_Java_Dev/Exercise Files/Ch02/02_03/agePrivileges.java
import java.util.*;
public class agePrivileges
{
 public static void main (String [] args)
 {
   //greater than 12 AND less than 20 - teen . 
   // else you are an adult or a child
   // less than 20 or !A US Citizen - you don't get to vote
   //else you get to vote
   Scanner scan = new Scanner (System.in);
   System.out.println("Please enter your age");
   int age = scan.nextInt();
   System.out.println("Are you a US Citizen?(Enter Yes or No)");
   String citizen = scan.next();
   if ((age >12) && (age <20))
     System.out.println ("You are a teen. You get to throw tantrums!");
   else
     System.out.println("You are a child. You get to throw tantrums!");
   if (( age < 20) || (!citizen.equalsIgnoreCase ("Yes")) )   
     System.out.println ("Sorry! No voting privileges");
   else
     System.out.println("You are an adult and you get to vote!");
 }
}

File: Python Projects/Ex_Files_Python_Basics_Java_Dev/Exercise Files/Ch02/02_03/agePrivileges.py
print ("Please enter your age")
age = int (input ())
print ("Are you a US Citizen?")
citizen = bool (input ())

if (age > 12) and (age < 20):
	print ("You are a teen! You can throw tantrums.") 
else:
	print ("You are a child! You can also throw tantrums")

if (age <21) or (citizen == False):
	print ("Sorry no voting privileges")
else:
	print ("You are an adult and you get to vote!")
